lib/expansion_tile_card.dart

```diff
@@ -254,13 +272,14 @@ class _ExpansionTileCardState extends State<ExpansionTileCard> with SingleTicker
                   child: Padding(
                     padding: const EdgeInsets.all(2.0),
                     child: ListTile(
+                      isThreeLine: widget.isThreeLine,
                       contentPadding: widget.contentPadding,
                       leading: widget.leading,
                       title: widget.title,
                       subtitle: widget.subtitle,
-                      trailing: widget.trailing ?? RotationTransition(
+                      trailing: RotationTransition(
                         turns: _iconTurns,
-                        child: const Icon(Icons.expand_more),
+                        child: widget.trailing ?? Icon(Icons.expand_more),
```

Comment:
This would be an unexpected change in behavior from the original `ExpansionTile` that this widget is based on. I'd like to mirror the original Flutter version as much as possible.

Reply:
My thought was that you wouldn't have to do the RotationTransition yourself if it was this way, but I see what you mean. Do you think I should just revert this commit?

Reply:
Probably. Another option, since I could see why developers may want this, would be to have the rotation disabled by default if `trailing` is set and enabled with a new boolean. Not sure how well that would work though.